Выполнила ДЗ №7

alisaaniskina opened the pull request.

Commit:
Добавила файлы теста

Добавлен тест для проверки изменения имени и фамилии пользователя

Commit:
Изменила файл main_f.py

skyline-gleb commented on main_f.py:
```diff
@@ -0,0 +1,53 @@
+        """
+        driver = self.driver
+        driver.find_element_by_class_name("avatar-wrapper").click()
+        driver.implicitly_wait(5)
```
ая-яй, с этим не приму

skyline-gleb commented on main_f.py:
```diff
@@ -0,0 +1,53 @@
+        input_name[0].send_keys(self.name[0])
+        input_name[1].send_keys(self.name[1])
+        button_accept = driver.find_element_by_class_name("dbmodal-button").click()
+        driver.implicitly_wait(15)
```
да чтож такое))))

alisaaniskina replied:
это было написано до того, как ты рассказывал, просто исправить не успела, я обязательно перепишу )

skyline-gleb commented on requirements.txt:
```diff
@@ -0,0 +1 @@
+selenium
```
за это :+1:

skyline-gleb commented:
плохая практика перенастраивать драйвера, его impluiciti_wait. ставь сразу больше и всё. сейчас это в любом случае лучше

Commit:
Исправила main_f.py

Переделала команду ожидания элементов (перенесла настройку в setUp)
Изменила команду перехода по вкладкам

skyline-gleb commented:
:+1:
